src/renderer/components/shared/CanvasDataBuilder.js

```diff
@@ -132,6 +132,7 @@ const buildCommonInstanceNode = (instance, graqlVar, explanation, queryPattern)
   node.baseType = instance.baseType;
   node.var = graqlVar;
   node.attrOffset = 0;
+  node.offset = 0;
```

Comment:
hang on, what is this offset and why is it not set anymore?

Reply:
we no longer have the `offset` property on the node object. addressed in https://github.com/graknlabs/workbase/pull/304/commits/27f361b79d6deadfb1d482f13d17f92a9c5359b0